lib/pages/home_page.dart

```diff
@@ -13,17 +15,25 @@ class _HomePageState extends State<HomePage> {
   final GlobalKey<RefreshIndicatorState> _refreshIndicatorKey =
   new GlobalKey<RefreshIndicatorState>();
   int _currentIndex = 0;
+  static FirebaseUser _firebaseUser;
 
   final PageStorageBucket bucket = PageStorageBucket();
 
   //TODO: Replace for rest of pages
   List<Widget> _pages = [
     JobsPage(key: PageStorageKey('JobsPage')),
-    LoadingAccount(),
+    ChatList(firebaseUser: _firebaseUser),
```

Comment:
We should possibly add a key so that it doesn't refresh the page , maybe it doesn't by default